src/commands/config.py

```
@@ -0,0 +1,194 @@
+    if not (
+        interaction.user.guild_permissions.manage_guild
+        or interaction.user == interaction.guild.owner
+    ):
+        try:
+            if not is_admin().predicate(interaction):
+                await interaction.response.send_message(
+                    "You do not have permission to run this command.",
+                    ephemeral=True,
+                )
+                return
+        except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)